lib/elixir/lib/task.ex

```diff
@@ -225,6 +225,9 @@ defmodule Task do
 
   It contains these fields:
 
+    * `mfa` - a three-element tuple containing the module, function name,
+        and arity; stored for reflection purposes
```

Comment:
I'm  also wondering if we should add a mention like this:

> See `Task.async/1` and `Task.async/3` to see how this field is set.

Reply:
```suggestion
        and arity invoked to start the task in `async/1` and `async/3`
```

Reply:
That is prefect!